docs/testingplatform/pillars.md

```diff
@@ -11,5 +11,5 @@ The main driving factors for the evolution of the new testing platform are:
 * **Hostable**: The test runtime can be hosted in any .NET application. While a console application is commonly used to run tests, you can create a test application in any type of .NET application. This allows you to run tests within special contexts, such as devices or browsers, where there may be limitations.
 * **Support all .NET form factors**: Support current and future .NET form factors, including Native AOT.
 * **Performant**: Finding the right balance between features and extension points to avoid bloating the runtime with non-fundamental code. The new test platform is designed to "orchestrate" a test run, rather than providing implementation details on how to do it.
-* **Enough extensible**: The new platform provides essential extensibility points to allow for maximum customization of runtime execution. It allows you to configure the test process host, observe the test process, and consume information from the test framework within the test host process.
+* **Extensible enough**: The new platform is built on extensibility points to allow for maximum customization of runtime execution. It allows you to configure the test process host, observe the test process, and consume information from the test framework within the test host process.
```

Comment:
essential evokes to me that you cannot be without them, fundamental is probably more appropriate here, but imho is built on, communicates the point the best without any confusion.